Directory.Build.props

```diff
@@ -109,8 +109,6 @@
     <KeepNativeSymbols Condition="'$(KeepNativeSymbols)' == ''">false</KeepNativeSymbols>
     <!-- Used for launchSettings.json and runtime config files. -->
     <AppDesignerFolder>Properties</AppDesignerFolder>
-    <!-- Determine what architecture we are building on. -->
-    <BuildArchitecture Condition="'$(BuildArchitecture)' == ''">$([System.Runtime.InteropServices.RuntimeInformation]::ProcessArchitecture.ToString().ToLowerInvariant())</BuildArchitecture>
```

Comment:
Unless it's necessary for this change - I would prefer to do cleanup like changes in a follow-on PR. The PR is already large. And renames are tricky to review, so I would prefer to keep the PR as simple as possible.